net/relayd/src/opnsense/scripts/OPNsense/Relayd/post-install.php

```diff
@@ -0,0 +1,272 @@
+#!/usr/local/bin/php
```

Comment:
Using the built-in config migration would be better than keeping this post-install hook always

haproxy 1.x -> 2.0 uses the internal model migration facility for inspiration

Reply:
BaseModelMigration is cool :+1: 
But the old plugin has no version information in its model. Wait, it does not even have a model. :wink: 
The next versions from 2.0 on will have it, promised. :smiley: